transaction_integration_test.go

```diff
@@ -765,7 +768,16 @@ func TestTransactionSkipAdvancedFraudChecks(t *testing.T) {
 		t.Fatal(err)
 	}
 	if tx2.RiskData != nil {
-		t.Fatal("expected tx2.RiskData to be empty")
+		// https://developers.braintreepayments.com/guides/advanced-fraud-tools/server-side/java#response-handling
+		// quote: We return the risk data on all credit card transactions
+		// thus I expect we will receive a risk data struct
+		// but this struct should tell us the risk as NOT been evaluated...
+		// quote: The possible values of the risk decision are Not Evaluated, Approve, Review, and Decline
+		if tx2.RiskData.Decision != RiskDataNotEvaluated {
+			t.Fatalf("expected tx2.RiskData.Decision to be %q, got %q", RiskDataNotEvaluated, tx2.RiskData.Decision)
+		}
+	} else {
+		t.Fatal("expected tx2.RiskData to be set with Decision field to 'Not Evaluated' but was nil")
 	}
 }
```

Comment:
Can we add tests that test a 3DS transaction similar to in #170 here `TestTransaction3DSCreateTransactionAndSettleSuccess`, but using the logic in the Java SDK mentioned here https://github.com/lionelbarrow/braintree-go/pull/170/files#r123081324?